scss/abstracts/_reset.scss

```diff
@@ -0,0 +1,16 @@
+@use 'variables' as *;
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-family: 'Inter', sans-serif;
+    font-size: 10px;
```

Comment:
This is a root element, rem and em units will be referring to the font-size that's set here (em indirectly, since it's looking for it's parent elements' font-size). I would set it up as standard 16px and made sure rest of the code works around it. 10px is a small font-size, and setting this value up here, means you will have to scale up your font-sizes throughout rest of the files, even in areas where you wouldn't normally need to. In the future if you want to manipulate how big the text is, for example for accessibility reasons, you can do it by changing this value.

Reply:
Ok, thank you for claryfing that. I'll adjust the project to refer to 16px as a base.